src/components/date-range-picker/date-localization.pipe.ts

```diff
@@ -0,0 +1,16 @@
+import { Inject, LOCALE_ID, Pipe, PipeTransform } from '@angular/core';
+import { formatDate } from '@angular/common';
+import { DateLocalizationFormatter } from './date-localization-formatter.type';
+
+@Pipe({
+    name: 'dateLocale'
+})
+export class DateLocalizationPipe implements PipeTransform {
+
+    constructor(@Inject(LOCALE_ID) private locale: string) {}
```

Comment:
@ashley-hunter Does this depend on the consumer using Angular i18n? In the documentation site  on my environment this has the value "en-US" which is apparently the default if not configured.

Reply:
There is a `navigator.language` property available in the browser perhaps we should use instead (this is fairly new so I'm not sure if it works in IE 11 or not). But we could use the `LOCALE_ID` as fallback if IE is not supported

Reply:
Yes that sounds good. If that's not suitable then they still have the option of providing a function.